examples/resources/oidc-connector.yaml

```diff
@@ -1,52 +1,20 @@
 #
 # Example resource for an OIDC connector
-# We recommend using OIDC for G Suite, Auth0 and Keycloak
+# We recommend using oidc for G Suite, Auth 0 and Keyclock
```

Comment:
This line needs to be reverted to what it was before
`oidc` -> `OIDC`
`Auth 0` -> `Auth0`
`Keyclock` -> `Keycloak`

Reply:
This was pull from the UI, so I'll also track this issue in another ticket.